templates/CSI/cephfs/secret.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ csi_cephfs_secret | default ('csi-cephfs-secret') }}
+  namespace: {{ csi_cephfs_secret_ns | default('default') }}
+data:
+  # Required if provisionVolume is set to false
+  # To get base64 encoded username ( default - kubernetes): echo -n kubernetes|base64
+  # To get base64 encoded password ( default - kubernetes): ceph auth get-key client.kubernetes|base64
+  userID: {{ base64_encoded_user | default('a3ViZXJuZXRlcw==') }}
```

Comment:
Can we comment it if we not we need to create  kubernetes user else pvc will remain in Pending

Reply:
agree. and it definitely requires presence of a kubernetes user. Also, yesterday we faced some issue in mounting a pvc which was created with "kubernetes" user enabled. We will retest and raise issue.

I would need confirmation from other reviewers before commenting it